modules/dxp/apps/scim/scim-rest-impl/src/main/java/com/liferay/scim/rest/internal/manager/UserManagerImpl.java

```diff
@@ -675,6 +1002,68 @@ private com.liferay.portal.kernel.model.User _updatePortalUser(
 		return portalUser;
 	}
 
+	private UserGroup _updateUserGroup(long companyId, Group group)
+		throws Exception {
+
+		long groupId = GetterUtil.getLong(group.getId());
+
+		UserGroup userGroup = _getUserGroup(
+			companyId, group.getExternalId(), groupId);
+
+		if (userGroup == null) {
+			throw new NotFoundException(
+				"No group found with group ID " + groupId);
+		}
```

Comment:
`_getUserGroup()` will never return null? It will throw NotFoundException so this block is redundant?